pkg/aws/service.go

```diff
@@ -262,7 +262,7 @@ func Construct(cfg Config) (types.Service, error) {
 		indexesClient = telemetry.InstrumentRedisClient(indexesClient)
 	}
 
-	cachingQueue := NewSQSCachingQueue(cfg.Config, cfg.SQSCachingQueueURL, cfg.CachingBucket)
+	cachingQueue := NewSQSCachingQueue(cfg.Config, cfg.SQSCachingQueueID, cfg.CachingBucket)
```

Comment:
Why has URL been renamed to ID? I think it's maybe a bit confusing if this is a URL...even if it is the identifier for the queue.